foxy/distribution.yaml

```diff
@@ -596,6 +596,38 @@ repositories:
       url: https://github.com/CLOBOT-Co-Ltd/clober_msgs.git
       version: foxy-devel
     status: developed
+  clpe_ros:
+    doc:
+      type: git
+      url: https://github.com/canlab-co/clpe_ros.git
+      version: main
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/canlab-co/clpe_ros-release.git
+      version: main
```

Comment:
We should just remove the `release` entry, because there are no release artifacts at that URL.
```suggestion
```